### DISCUSSION TITLE
Aktuelle Höhenangabe anzeigen

### DISCUSSION THREAD

**Discussion by Woidera** [organicmaps/organicmaps#5743] [Ideas]
Bitte die aktuelle Höhenangabe auf der Kartenansicht anzeigen.

**Comment by biodranik** [comment]
You can select your current position (blue arrow) to see the current speed and elevation. 

Sie können Ihre aktuelle Position (blauer Pfeil) auswählen, um die aktuelle Geschwindigkeit und Höhe anzuzeigen.

**Comment by termorab** [comment]
Hi folks,
thanks for making such a great job.
I must agree to @Woidera. Showing the high on map while navigating would be appreciated. The information about the actual high is only available in planning mode, not while navigation. Seen on android phone and tablet.
Please make it happen 🙏

**Comment by CanRau** [comment]
I'm curious if it'd be possible to show altitude of any point I tap on the map? Especially when I have contour lines enabled it feels like the data is kinda already there tho I have no idea how detailed the data actually is and maybe doesn't provide for that?

**Reply by biodranik** [reply]
Only track/roads points and elevation curves has elevation data. It can be interpolated in theory… A better elevation model that allows to get elevation at any point more precisely, and show hill shadings would be better of course.
